tests/phpunit/tests/admin/includesListTable.php

```diff
@@ -389,6 +419,8 @@ function() {
 
 	/**
 	 * @ticket 45089
+	 *
+	 * @covers WP_Posts_List_Table::get_sortable_columns
```

Comment:
```suggestion
	 * @covers WP_Posts_List_Table::print_column_headers
```
The test isn't actually running the  code in`get_sortable_columns` but rather is mocking it and ensuring that it is invoked. What is being tested? That `print_column_headers` renders columns that are sortable where "sortable" is a condition. Also noting that `get_sortable_columns` is a protected method used to configure the columns being rendered.